### PR TITLE
Dev cleanup: fixing various issues

### PR DESCRIPTION
**Pull request by esciara**

### CODE REVIEW

**Comment by esciara** [comment]
Beware of false negative on tests: see #131.

File: semantic_release/__init__.py [hunk 1]
```diff
@@ -3,10 +3,6 @@
 __version__ = '4.1.1'
 
 
-from .errors import UnknownCommitMessageStyleError  # noqa; noqa
```
Comment (relekang) [inline semantic_release/__init__.py]: Why are these removed? This is a breaking change, since this will break imports of errors directly from `semantic_release`. Can you please add them back?

Comment (esciara) [reply semantic_release/__init__.py]: Yes, I removed them as these caused flake8 errors (see #129), and tests are passing when removing these imports. So it is not clear what pb this causes.

Can you clarify the pb with an actual test or specific example?

Comment (esciara) [reply semantic_release/__init__.py]: hi @relekang ,

A kind reminder. Could you clarify please? Would love to work on having this PR merged.

Comment (relekang) [reply semantic_release/__init__.py]: Anyone who has installed this package can have done `from semantic_release import UnknownCommitMessageStyleError` and next release after this will break this so thus we need a major release in order to do it.
